RFS/src/test/java/com/rfs/integration/EndToEndTest.java

```diff
@@ -1,67 +1,117 @@
 package com.rfs.integration;
 
+import java.io.File;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+import org.hamcrest.Matchers;
 import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.io.TempDir;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ArgumentsSource;
 
+import com.rfs.common.ClusterVersion;
+import com.rfs.common.FileSystemRepo;
+import com.rfs.common.FileSystemSnapshotCreator;
+import com.rfs.common.OpenSearchClient;
+import com.rfs.framework.ClusterOperations;
 import com.rfs.framework.SearchClusterContainer;
 import com.rfs.framework.SimpleRestoreFromSnapshot;
-
-import lombok.extern.slf4j.Slf4j;
+import com.rfs.transformers.TransformFunctions;
+import com.rfs.version_es_6_8.SnapshotRepoProvider_ES_6_8;
+import com.rfs.version_es_6_8.GlobalMetadataFactory_ES_6_8;
+import com.rfs.version_es_6_8.IndexMetadataFactory_ES_6_8;
+import com.rfs.version_os_2_11.GlobalMetadataCreator_OS_2_11;
+import com.rfs.version_os_2_11.IndexCreator_OS_2_11;
+import com.rfs.worker.IndexRunner;
+import com.rfs.worker.MetadataRunner;
+import com.rfs.worker.SnapshotRunner;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
 
 /**
  * Tests focused on setting up whole source clusters, performing a migration, and validation on the target cluster
  */
-@Slf4j
 public class EndToEndTest {
```

Comment:
How would you feel about adding these to the "slow tests"?  E.g. use the `@Tag("longTest")` annotation here.